src/components/layout/web/Footer.tsx

```diff
@@ -47,6 +47,11 @@ export default function Footer() {
                   Politique de confidentialité
                 </Link>
               </FooterLink>
+              <FooterLink>
+                <Link href={`mailto:${process?.env?.CONTACT_EMAIL}`} title='Nous contacter'>
```

Comment:
```suggestion
                <Link href={`mailto:${process.env.CONTACT_EMAIL}`} title='Nous contacter'>
```

Reply:
manque un target _blank et les rel

Reply:
- pour les "rel" : il s'agit d'un mailto, ce n'est pas certain que ça soit requis https://stackoverflow.com/questions/51193862/mailto-in-href-should-i-add-rel-noopener-noreferrer
- pour le target _blank : pareil, j'ai un doute sur l'utilité dans un lien mailto : https://www.badsender.com/en/2023/01/27/target-blank-links-email/

Reply:
dans notre code, il y avait précédemment 2 cas : un avec target _blank et "rel", l'autre non...